Build-and-Deploy-Your-First-Dashboard.mdx

```diff
@@ -0,0 +1,300 @@
+
+
+### Build your first chart
+
+It requires four steps to build your first chart:
+
+<AccordionGroup>
+  <Accordion title="1. Select raw table(s)">
+
+    Search and select the **aggregations** table. 
+
+    <Frame>
+<img
+  className="block dark:hidden"
+  src="/images/build-your-first-dashboard/light/select-raw-tables.png"
+  alt="Hero Light"
+/>
+<img
+  className="hidden dark:block"
+  src="/images/build-your-first-dashboard/dark/select-raw-tables-dark.png"
+  alt="Hero Dark"
+/>
+</Frame>
+    <Tip>
+
+    The list of available raw table(s) is generated after the [data connection](/data-connection/data-connection) is set up.
+
+    </Tip>
+
+  </Accordion>
+
+{" "}
+<Accordion title="2. Prepare data">
+Click on the **+ aggregations** , this will create ```SQL SELECT * FROM "aggregations" ```   
+in the SQL editor.
+
+Click on **Preview data** to run the SQL and **Next** once you see the preview data.
+     <Frame>
+
+  <img
+  className="block dark:hidden"
+  src="/images/build-your-first-dashboard/light/query-builder.png"
+  alt="Hero Light"
+/>
+<img
+  className="hidden dark:block"
+  src="/images/build-your-first-dashboard/dark/query-builder-dark.png"
+  alt="Hero Dark"
+/>
+    </Frame>
+
+
+ <Tip>
+    If you are more advanced with SQL, you can add more tables and customise your SQL.
+    </Tip>
+  </Accordion>
+
+{" "}
+<Accordion title="3. Build chart">
+  
+  For your first chart select a **Grouped bar**, and choose **country** as the x-axis and  **demand** as the y-axis.
+  the corresponding x- & y-axis
+  
+  For future charts see **BROKEN LINK** [Chart Configuration](/content/charts/area-line-chart-a) for all chart settings. 
+  
+  
+
+     <Frame>
+
+  <img
+  className="block dark:hidden"
+  src="/images/build-your-first-dashboard/light/chart-builder-dark.png"
+  alt="Hero Light"
+/>
+<img
+  className="hidden dark:block"
+  src="/images/build-your-first-dashboard/dark/chart-builder-dark.png"
+  alt="Hero Dark"
+/>
+     </Frame>
+
+</Accordion>
+
+  <Accordion title="4. Configure filters">
+    You can set up filters by clicking on **configure filters** and selecting the relevant column and filter type.
+    ![configure-filters](/images/filters/filters-guide-4.png)
+
+  Click **Save and add to dashboard** when you are happy with your chart, this will save the chart. You could also
+  create a copy of the chart by clicking **Duplicate**.
+    <Tip>
+    For a more comprehensive overview, check out our [in-depth filtering guide](/filters/filters-in-depth).
+    </Tip>
+    
+  </Accordion>
+</AccordionGroup>
+
+### Configure your dashboard
+
+You are able to customise your dashboard by adding dashboard filters, controlled filters and additional charts. 
+
+To resize charts you can simply drag the chart to the size you would like.
+
+To move a chart simply drag and drop the chart.
+
+#### Add dashboard filters
+
+To add a dashboard filter please click on **New Filter**
```

Comment:
This isn't enough, let's at least link tot he dashboard filter page but better yet walk through concretely 1 filter setuo